docs/tutorials/cloud_fit_deploy/cloud-aws-sagemaker-deployment.md

```diff
@@ -36,6 +36,9 @@ import numpy as np
 def model_fn(model_dir):
```

Comment:
Could we add `# from autogluon.multimodal import MultiModalPredictor` in the imports?